docs/source/receptorctl/receptorctl_work_submit.rst

```diff
@@ -0,0 +1,39 @@
+-----------
+work submit
+-----------
+
+.. contents::
+   :local:
+
+``receptorctl work submit`` submits a new unit of work.
+
+Command syntax: ``receptorctl --socket=<socket_path> work submit [<<Options>>] <<WorkType>> [<<Command Parameters>>]``
+
+``socket_path`` is the control socket address for the Receptor connection.
+   The default is ``unix:`` for a Unix socket.
+   Use ``tcp://`` for a TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.
+
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``WorkType`` is the execution request type for the work unit.
+
+``Command Parameters`` are parameters passed to the work command.
+
+``-a``, ``--param TEXT`` adds additional Receptor parameter in key=value format.
+``-f``, ``--follow`` causes Receptorctl to remain attached to the job and print its results.
+``-l``, ``--payload-literal TEXT`` uses the ``TEXT`` as the literal unit of work data.
+``-n``, ``--no-payload`` sends an empty payload.
+``--node <<Node ID>>`` is the Receptor node to run the work on. It defaults to the local node.
+``-p``, ``--payload TEXT`` specifies the file containing unit of work data. Use - for stdin.
+``--rm`` releases the work unit after completion.
+``--signwork`` digitally signs remote work submissions to stdout.
+``--tls-client TEXT`` specifies the TLS client used when submitting work to a remote node.
+``--ttl TEXT`` specifies the time to live until remote work must start, e.g. 1h20m30s or 30m10s.
```

Comment:
```suggestion
``--ttl TEXT`` specifies the time to live until remote work must start, e.g. 1h20m30s or 30m10s.
```
```suggestion
``--ttl <<TEXT>>`` specifies the time to live (TTL) for remote work requests in ``##h##m##s`` format; for example ``1h20m30s`` or ``30m10s``.
```

Reply:
Assuming TTL here is the lifespan, what happens when that value gets exceeded? Does the request fail? Is there an error message somewhere?

Reply:
I'm being kind of lazy here and not setting things up to find out for myself - just going to ask the question. But does this add a new unit of work to a processing queue on the Receptor node?

If so, is there a way for users to find out what other work items are already in the queue and use that information to determine the best TTL value?

Also, does Receptor run jobs concurrently? Say there is a thread per job, what performance considerations are there?

Reply:
> I'm being kind of lazy here and not setting things up to find out for myself - just going to ask the question. But does this add a new unit of work to a processing queue on the Receptor node?
Yes, it will add a new unit of work on the local node and a separate unit of work on the remote node.
> 
> If so, is there a way for users to find out what other work items are already in the queue and use that information to determine the best TTL value?
Yes for the other work items (receptorctl work list).

The TTL question is interesting but some work would need to be done to suggest an approach.
> 
> Also, does Receptor run jobs concurrently? Say there is a thread per job, what performance considerations are there?
Yes, Receptor runs jobs concurrently on each execution plane node.  Some work would need to be done to suggest approach for performance considerations.

Reply:
Cool. Performance considerations are always tricky with things clustered across networks. There are so many factors that can impact performance.

It might be useful to give some general guidance like "do some evaluation in a test environment to get a feel for how long work units take to run under your specific conditions before you request work units in production". Maybe also something along the lines of "try to balance requests for units of work across Receptor nodes to ensure the workload is distributed as evenly as possible". Although that might be a bit too vague and come across as unhelpful in the docs. Finding the optimal number of work units per node really depends and is something users need to find for themselves - docs for that are well out of the scope of this PR so maybe we should just leave it. If you do want to have a stab at any of that stuff, I'm game.

Reply:
i would say leave it for now.  If needed, I would suggest a Performance doc